.github/workflows/build-test.yml

```
@@ -19,43 +19,546 @@ jobs:
+    with:
+      java: 17, 21
+      scala: 2.13.x, 3.x
+      gradle-build-root: play-java-chatroom-example
+      add-dimensions: >-
+        { "server": [ "pekko", "netty"], "build_tool": ["sbt", "gradle"] }
+      exclude: >-
+        [{ "server": "netty", "build_tool": "gradle" }]
+      cmd: |
+        ./test.sh --sample=play-java-chatroom-example --backend-server=$MATRIX_SERVER --build-tool=$MATRIX_BUILD_TOOL
+
+  java-compile-di:
+    name: "Java :: Compile DI"
+    uses: playframework/.github/.github/workflows/cmd.yml@v4
+    needs: changes
+    if: ${{ (github.event_name != 'pull_request' && !failure() && !cancelled()) || contains(fromJSON(needs.changes.outputs.samples), 'java-compile-di')}}
+    with:
+      java: 17, 21
+      scala: 2.13.x, 3.x
+      gradle-build-root: play-java-compile-di-example
+      add-dimensions: >-
+        { "server": [ "pekko", "netty"], "build_tool": ["sbt", "gradle"] }
+      exclude: >-
+        [{ "server": "netty", "build_tool": "gradle" }]
+      cmd: |
+        ./test.sh --sample=play-java-compile-di-example --backend-server=$MATRIX_SERVER --build-tool=$MATRIX_BUILD_TOOL
+
+  java-dagger2:
+    name: "Java :: Dagger2"
+    uses: playframework/.github/.github/workflows/cmd.yml@v4
+    needs: changes
+    if: ${{ (github.event_name != 'pull_request' && !failure() && !cancelled()) || contains(fromJSON(needs.changes.outputs.samples), 'java-dagger2')}}
+    with:
+      java: 17, 21
+      scala: 2.13.x, 3.x
+      gradle-build-root: play-java-dagger2-example
+      add-dimensions: >-
+        { "server": [ "pekko", "netty"], "build_tool": ["sbt", "gradle"] }
+      exclude: >-
+        [{ "server": "netty", "build_tool": "gradle" }]
+      cmd: |
+        ./test.sh --sample=play-java-dagger2-example --backend-server=$MATRIX_SERVER --build-tool=$MATRIX_BUILD_TOOL
+
+  java-ebean:
+    name: "Java :: EBean"
+    uses: playframework/.github/.github/workflows/cmd.yml@v4
+    needs: changes
+    if: ${{ (github.event_name != 'pull_request' && !failure() && !cancelled()) || contains(fromJSON(needs.changes.outputs.samples), 'java-ebean')}}
+    with:
+      java: 17, 21
+      scala: 2.13.x, 3.x
+      gradle-build-root: play-java-ebean-example
+      add-dimensions: >-
+        { "server": [ "pekko", "netty"], "build_tool": ["sbt", "gradle"] }
+      exclude: >-
+        [{ "server": "netty", "build_tool": "gradle" }]
+      cmd: |
+        ./test.sh --sample=play-java-ebean-example --backend-server=$MATRIX_SERVER --build-tool=$MATRIX_BUILD_TOOL
+        
+  java-fileupload:
+    name: "Java :: File Upload"
+    uses: playframework/.github/.github/workflows/cmd.yml@v4
+    needs: changes
+    if: ${{ (github.event_name != 'pull_request' && !failure() && !cancelled()) || contains(fromJSON(needs.changes.outputs.samples), ' java-fileupload')}}
+    with:
+      java: 17, 21
+      scala: 2.13.x, 3.x
+      gradle-build-root: play- java-fileupload-example
```

seems like you copy pasted a space:
```suggestion
      gradle-build-root: play-java-fileupload-example
```

---

So stupid find/replace mistake 🤦 Thank you, Matthias!